app/src/main/res/layout/activity_download_webpage_task.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I know I asked William about this and he said you were using all ConstraintLayouts. If you were to use a RelativeLayout, you would be able to change the screen orientation when you rotate the phone. Up to you.

Reply:
I'm actually not sure that this needs a layout at all - the DownloadWebpageTask is just a wrapper to get sheets data asynchronously. I think I can delete the layout file without issue. I'll try it tomorrow.